contentcuration/contentcuration/frontend/channelEdit/views/TreeView/TreeViewBase.vue

```diff
@@ -414,7 +413,7 @@
         this.noSyncNeeded = true;
         this.showProgressModal = true;
       },
-      deleteChannel() {
+      deleteChannelModal() {
```

Comment:
`deleteChannel` also happened to be the name of a vuex action that was mapped into this component, so these were conflicting.